docs/install/local-docker.md

```diff
@@ -0,0 +1,83 @@
+# Install MLRun locally using Docker
+
+You can install and use MLRun and Nuclio locally on your computer, this will not include all the services and elastic 
+scaling capabilities which you can get with the Kubernetes based deployment but is much simpler to start with.
```

Comment:
You can install and use MLRun and Nuclio locally on your computer. This does not include all the services and elastic scaling capabilities, which you can get with the Kubernetes based deployment, but it is much simpler to start with.